src/test/java/com/ipcounter/classes/IpConverterTest.java

```diff
@@ -0,0 +1,53 @@
+package com.ipcounter.classes;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class IpConverterTest {
+
+  @Test
```

Comment:
btw: just wondering why 4th version of junit? And seems that parametrized tests can come in handy here :)

Reply:
I want to know what part broke, so I prefer full names for my test cases